thetis/solver.py

```diff
@@ -618,3 +650,4 @@ def iterate(self, update_forcings=None, update_forcings3d=None,
                         print '{0:25s} : {1:11.6f} {2:11.2f}'.format(
                             label, c, relcost)
                         sys.stdout.flush()
+                gc.collect()
```

Comment:
I would really like to track this down, as it must be an issue *somewhere*.  However, in the past I have had absolutely zero luck getting to the bottom of this kind of thing.